tests/cypress/integration/regTest_for_export_task/innerExportTask.js

```diff
@@ -0,0 +1,138 @@
+// Start writing your Cypress tests below!
+// If you're unfamiliar with how Cypress works,
+// check out the link below and learn how to write your first test:
+// https://on.cypress.io/writing-first-test
+
+/// <reference types="cypress" />
+
+Cypress.Commands.add('exportInnerTask', ({
+    taskName,
+    archiveCustomName,
+    order,
+    imageClick = false,
+}) => {
+    const orderId = Number.isNaN(parseInt(order, 10)) ? 0 : parseInt(order, 10);
```

Comment:
```suggestion
    const orderId = +order || 0; 
```

But generally, if we need an integer here, passing a string to this command does not make sense